### PR TITLE
README: Noob friendly instructions :)

### PR DESCRIPTION
**Pull request by akshay-49**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-![Explosion!!](https://imgur.com/download/FPZPLK9)
+A fork of [YetAnotherSMSBomber](https://github.com/AvinashReddy3108/YetAnotherSMSBomber) with instructions for newbies and a few changes
```
Comment (AvinashReddy3108) [inline README.md]: Revert this, obviously.

File: README.md [hunk 2]
```diff
@@ -69,10 +140,11 @@ python3 bomber.py -p -N 500 -c 91 -T 30 -t 20 <TARGET>
 ```
 
 ## Credits and Thanks
-- Huge kudos to [iMro0t](https://github.com/iMro0t) for the original source code. Find it [here](https://github.com/iMro0t/bomb3r/).
```
Comment (AvinashReddy3108) [inline README.md]: The credits must remain.

File: README.md [hunk 3]
```diff
@@ -69,10 +140,11 @@ python3 bomber.py -p -N 500 -c 91 -T 30 -t 20 <TARGET>
...
-- Thanks [botallen](https://github.com/botallen) for the recent fixes which have been merged from the original repo.
-- [SpeedX](https://github.com/TheSpeedX)'s [TBomb](https://github.com/TheSpeedX/TBomb) for some API's used here.
-- [fonic](https://github.com/fonic) for his awesome formatter for `argparse`. Check it out in this [gist](https://gist.github.com/fonic/fe6cade2e1b9eaf3401cc732f48aeebd)!
 
-## License
-This project is licensed under the [GNU General Public License v3.0](https://github.com/AvinashReddy3108/YetAnotherSMSBomber/blob/master/LICENSE)
+All credits go to [AvinashReddy3108](https://github.com/AvinashReddy3108) you can find his repo [here](https://github.com/AvinashReddy3108/YetAnotherSMSBomber)
```
Comment (AvinashReddy3108) [inline README.md]: These lines must go, except the Windows instructions part.

Comment (akshay-49) [reply README.md:144]: You mean add all the credits and remove the windows part?

File: README.md [hunk 4]
```diff
@@ -1,25 +1,96 @@
...
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+
+# Install Dependencies
+brew install git
+brew install python3
+sudo easy_install pip
+sudo pip install --upgrade pip
+
+# Clone this repo
+git clone https://github.com/akshay-49/SimpleSMSBomber.git
```
Comment (AvinashReddy3108) [inline README.md]: change the clone URL wen?

Comment (akshay-49) [reply README.md]: Forgot that one will change.

**Comment by niteshraj2310** [comment]
👀

**Pull request merged**